test/extended/builds/pipeline.go

```diff
@@ -200,7 +201,7 @@ var _ = g.Describe("[builds][Slow] openshift pipeline build", func() {
 			br.AssertSuccess()
 
 			g.By("confirm all the log annotations are there")
-			_, err = jenkins.ProcessLogURLAnnotations(oc, br)
+			err = jenkins.ProcessLogURLAnnotations(oc, br, !useSnapshotImage)
```

Comment:
ah.  I assumed we were testing both versions and the usesnapshot was just telling us whether to use the local image for each, or pull them.

Seems like we might as well change the PR test to use the v2 image at this point.